### PR TITLE
Mix - basic transaction creation

### PR DESCRIPTION
**Pull request by yann300**
Allow to create basic tr and not only contract call.

### CODE REVIEW

File: mix/ClientModel.cpp [hunk 1]
```diff
@@ -255,7 +255,10 @@ void ClientModel::setupState(QVariantMap _state)
 		u256 value = (qvariant_cast<QEther*>(transaction.value("value")))->toU256Wei();
 		u256 gasPrice = (qvariant_cast<QEther*>(transaction.value("gasPrice")))->toU256Wei();
 		QString sender = transaction.value("sender").toString();
-		bool isStdContract = (transaction.value("stdContract").toBool());
+		bool isStdContract = transaction.value("stdContract").toBool();
+		bool isContractCreation;
```
Comment (arkpar) [inline mix/ClientModel.cpp]: Can be left unitialized

File: mix/ClientModel.cpp [hunk 2]
```diff
@@ -307,6 +308,14 @@ void ClientModel::executeSequence(vector<TransactionSettings> const& _sequence,
 			m_gasCosts.clear();
 			for (TransactionSettings const& transaction: _sequence)
 			{
+				QString contractName = resolveContractName(transaction.contractId);
+				QString address = resolveToken(transaction.contractId, deployedContracts);
+				if (transaction.functionId == "(transfert)")
```
Comment (arkpar) [inline mix/ClientModel.cpp]: Can we make it without a magic string constant? Using empty string or explicit flag

File: mix/qml/TransactionDialog.qml [hunk 3]
```diff
@@ -96,9 +114,7 @@ Dialog {
 				functionsModel.append({ text: functions[f].name });
 			}
 		}
-		//append constructor
-		functionsModel.append({ text: contractId });
-
+		functionsModel.append({ text: "(transfert)" });
```
Comment (arkpar) [inline mix/qml/TransactionDialog.qml]: "None" would be better

Comment (yann300) [inline mix/qml/TransactionDialog.qml]: done

**Pull request merged**